### PR TITLE
Add V4 upgrade tutorial

### PR DESCRIPTION
**Pull request by TarekAwwad**

### CODE REVIEW

File: Testnet/kichain-t-4/UPGRADE_V4.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# check version - should be Testnet-4.0.0-beta (commit 9627c0de235a505bd85fc6daaaa2c049d3920a31)
+./kid version --long
+
+# make a directory if you haven't already
+mkdir -p $DAEMON_HOME/cosmovisor/upgrades/v4/bin
+
+# copy this new binary
+cp kid $DAEMON_HOME/cosmovisor/upgrades/v4/bin/
+
+# this should return Testnet-3.0.0-beta
```
Comment (WinterNis) [inline Testnet/kichain-t-4/UPGRADE_V4.md]: This should be Testnet-4.0.0-beta

File: Testnet/kichain-t-4/UPGRADE_V4.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+We provide static binary releases with verifiable checksums. We encourage validators to build these binaries themselves using our dedicated tutorial and check the resulting checksum.
+
+The client version should be:
+```bash
+kid version --long
+
+# name: kitools
+# server_name: kid
+# version: Testnet-4.0.0-alpha.1
```
Comment (WinterNis) [inline Testnet/kichain-t-4/UPGRADE_V4.md]: This should be Testnet-4.0.0-beta

Comment (TarekAwwad) [reply Testnet/kichain-t-4/UPGRADE_V4.md]: 👍

File: Testnet/kichain-t-4/UPGRADE_V4.md (hunk 1 above)
Comment (TarekAwwad) [reply Testnet/kichain-t-4/UPGRADE_V4.md]: 👍

File: Testnet/kichain-t-4/UPGRADE_V4.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# Download new binary (amd64 version, please adapt if you use an arm arch)
+wget https://github.com/KiFoundation/ki-tools/releases/download/4.0.0-beta/kid-testnet-4.0.0-beta-linux-amd64
+mv kid-testnet-4.0.0-beta-linux-amd64 kid
+chmod +x kid
+
+# check shasum - should be 172164f58005b362dfac754a1251bf61b6274160ffc5ea13da6525407645e2f3
+sha256sum ./kid
+
+# check version - should be Testnet-4.0.0-beta (commit 9627c0de235a505bd85fc6daaaa2c049d3920a31)
```
Comment (judotki) [inline Testnet/kichain-t-4/UPGRADE_V4.md]: commit is `b0ea200bd7a3f4a85a65d20694d07d0905a9638a`

**Review by WinterNis — APPROVED** [review]

**Pull request merged**